xray/cli.go

```diff
@@ -177,7 +177,11 @@ func auditMvnCmd(c *cli.Context) error {
 	if err != nil {
 		return err
 	}
-	xrAuditMvnCmd := audit.NewAuditMvnCommand().SetInsecureTls(c.Bool(cliutils.InsecureTls)).SetServerDetails(serverDetailes).
+	format, err := getXrayPrintFormat(c)
+	if err != nil {
+		return err
+	}
+	xrAuditMvnCmd := audit.NewAuditMvnCommand().SetInsecureTls(c.Bool(cliutils.InsecureTls)).SetServerDetails(serverDetailes).SetPrintFormat(format).
```

Comment:
SetPrintFormat --> SetOutputFormat

(let's change for the other commands as well)